web_src/css/modules/label.css

```diff
@@ -4,7 +4,7 @@
 .ui.label {
   display: inline-flex;
   align-items: center;
-  gap: .25rem;
+  gap: var(--gap-label);
```

Comment:
Sorry but I don't think it should use `gap-label` for "label inner content" and "gap between labels"

The only "gap" variables we need are for 2 cases:

1. For inline text (see the flex-text-inline, 0.25em)
2. For block text (see the flex-text-block, 0.5em)

So the gap depends on what layout the element is used for.

For a `.ui.label`, by default it has `inline-flex`, so it should use the same gap as flex-text-inline.


For "labels-list" gap, it depends on the context, for inline context, so 0.25em too.

-----

In short, we should drop this `var(--gap-label)`

Reply:
Seems alright, but I don't imagine we'd ever make those two vars have different values.

Reply:
Yep, for `.ui.label` itself, itself is an inline element; for "labels-list", when it is used in a sentence, it is also in an inline context.